examples/lab-robust/v2/qwiklabs.yaml

```diff
@@ -5,11 +5,22 @@ title: Robust Lab Example
 description: Seriously the best lab you've ever taken. Bar none.
 default_locale: en
 duration: 60
-level: easy
+max_duration: 60
+credits: 5
+level: fundamental
+logo: source-code.png
 tags:
 - sample
 - life-changing
 - gcp
+product_tags: ['app engine', 'kubernetes engine', 'cloud functions']
+role_tags: ['cloud architect', 'developers backend', 'developers frontend']
+domain_tags:
+- infrastructure
+- application development
+- data
+legacy_display_options:
```

Comment:
Just double-checking without understanding how we handle it - this sounds like a property that we put to old labs for some legacy reasons, I'm not sure we would want to include this in the new labs and hence not sure if that's sth we want to promote here. Have you found the meaning of this property?

Reply:
- For legacy_display_options last time we used this attribute from [4 years ](https://github.com/CloudVLab/qwiklabs-test-content/blob/ff5271ea63f5fa1a8b943d4c87e8ab32690ad861/labs/test_v1_lab_migration_5/qwiklabs.yaml#L10)
this [Cl](https://critique.corp.google.com/cl/258640416/depot/google3/cloud/training/qwiklabs/content_bundle/schema/content_bundle_lab.yaml) for that
Should i remove it?

Reply:
The only usages are in test repos and in archived labs: 
https://github.com/search?q=org%3ACloudVLab+legacy_display_options+-repo%3ACloudVLab%2Fqwiklabs-test-content+-repo%3ACloudVLab%2Fqwiklabs-content-bundle-spec+&type=code

Please add a buganizer issue to remove this property entirely.